Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ruby:2.2.2
+
+RUN apt-get update
+RUN apt-get install -y wget apt-transport-https
```

Comment:
these should be 1 line or you end up in a place where the update is cached and the install fails